components/MessageOverlay.tsx

```diff
@@ -125,5 +121,5 @@ export interface MessageOverlayProps {
   onRetry?: () => void;
   onBackdropPress?: () => void;
   onShow?: () => void;
-  customHeight?: number | string | undefined;
+  minHeight?: number | string | undefined;
```

Comment:
We are passing customHeight prop from other components as well, would this prop name change affect there? Can we rename in consumers as well?

Reply:
Changed on all the other consumers